scripts/src/java/org/oppia/android/scripts/AccessibilityLabelCheck.kt

```diff
@@ -0,0 +1,94 @@
+package org.oppia.android.scripts
+
+import org.w3c.dom.Node
+import org.w3c.dom.NodeList
+import java.io.File
+import java.util.stream.Collectors
+import java.util.stream.IntStream
+import javax.xml.parsers.DocumentBuilderFactory
+
+/**
+ * Script for ensuring that all the Activities the
+ * repo are defined with accessibility labels.
+ */
+class AccessibilityLabelCheck {
+  companion object {
+    @JvmStatic
+    fun main(vararg args: String) {
+      val repoPath = args[0] + "/"
+
+      val manifesFilePath = args[1]
+
+      val fullPathToManifestFile = repoPath + manifesFilePath
+
+      val accessibilityLabelCheck: AccessibilityLabelCheck = AccessibilityLabelCheck()
+
+      val builderFactory = DocumentBuilderFactory.newInstance()
+
+      val docBuilder = builderFactory.newDocumentBuilder()
+
+      val doc = docBuilder.parse(File(fullPathToManifestFile))
+      doc.getDocumentElement().normalize()
+
+      val completeActivityList = accessibilityLabelCheck.convertNodeListToListOfNode(
+        doc.getElementsByTagName("activity")
+      )
+
+      val activityListWithoutLabel = completeActivityList.filter { node ->
+        accessibilityLabelCheck.checkIfActivityHasMissingLabel(node)
+      }
+
+      accessibilityLabelCheck.logFailures(activityListWithoutLabel)
+
+      if (activityListWithoutLabel.size != 0) {
+        throw Exception(ScriptResultConstants.ACCESSIBILITY_LABEL_CHECK_FAILED)
+      } else {
+        println(ScriptResultConstants.ACCESSIBILITY_LABEL_CHECK_PASSED)
+      }
+    }
+  }
+
+  /**
+   * Checks whether a activity element has a missing label.
```

Comment:
`an` instead of `a`

`Checks whether an activity element has a missing label.`

Reply:
Done

Reply:
@rt4914 Can we resolve this now?